composer.json

```diff
@@ -53,13 +53,16 @@
 	"require-dev": {
 		"brianium/paratest": "^6.5",
 		"cweagans/composer-patches": "^1.7.3",
+		"jetbrains/phpstorm-stubs": "dev-master#8ea3bf44722f76a3a32c89fe769b6144c253f8a6",
```

Comment:
Latest version found also this miscategorization. Looks valid to me as it is used only in `ReflectionProviderGoldenTest`, no usage in src.

(but there is string reference in `compiler/src/Console/PrepareCommand.php`, might be worth checking)

Reply:
Reverted. [Reason](https://github.com/phpstan/phpstan-src/pull/2875#discussion_r1495886870)